projects/arlas-toolkit/assets/i18n/fr.json

```diff
@@ -38,13 +38,14 @@
 	"Remove": "Supprimer",
 	"Date": "Date",
 	"Name": "Nom",
-	"Add a dataset": "Ajouter un bookmark",
```

Comment:
I don't know if we should use the french word for bookmarks :/

Reply:
If we think that bookmark is a better choice, I can revert those changes !